pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -365,11 +367,15 @@ func (m *managerImpl) processShutdownEvent() error {
 			go func(pod *v1.Pod, group podShutdownGroup) {
 				defer wg.Done()
 
-				gracePeriodOverride := group.ShutdownGracePeriodSeconds
+				gracePeriodOverride := podDefaultTerminationPeriod
 
-				// If the pod's spec specifies a termination gracePeriod which is less than the gracePeriodOverride calculated, use the pod spec termination gracePeriod.
-				if pod.Spec.TerminationGracePeriodSeconds != nil && *pod.Spec.TerminationGracePeriodSeconds <= gracePeriodOverride {
-					gracePeriodOverride = *pod.Spec.TerminationGracePeriodSeconds
+				// If the pod's spec specifies a termination gracePeriod then use that grace period
```

Comment:
I feel like the old comment was more accurate. we're still overriding the period with the group one if the pod's is greater

Reply:
Yeah, good point. Reverted this line.